pkg/i18n/translate.nl.toml

```diff
@@ -0,0 +1,15 @@
+[EditConfig]
```

Comment:
how do translate.**.toml files compare to active.**.toml files?

Reply:
The "active"/"translate" convention is from goi18n. "active" contains the translated strings, "translate" contains strings to be translated.